actors/states/check.go

```diff
@@ -52,46 +54,33 @@ func CheckStateInvariants(tree *Tree, expectedBalanceTotal abi.TokenAmount, prio
 			if err := tree.Store.Get(tree.Store.Context(), actor.Head, &st); err != nil {
 				return err
```

Comment:
Returning an error loudly like this seems fine given how messed up the tree must be to not have a required singleton.  However I don't understand why we don't follow the new pattern with this class of errors. Are the effects on other actors invariant checks (i.e. in this case not being able to resolve id addresses) a complication that is hard to handle if we keep going? Or is just being loud because it is a very bad tree the point?

Reply:
There is no principled reason. I didn't change it only because the motivation to change it wasn't very strong.